libdnf5/base/solver_problems.cpp

```diff
@@ -63,7 +67,31 @@ static const std::map<ProblemRules, BgettextMessage> PKG_PROBLEMS_DICT = {
         " the following protected packages: {}")},
     {ProblemRules::RULE_PKG_REMOVAL_OF_RUNNING_KERNEL,
      M_("The operation would result in removing"
-        " of running kernel: {}")}};
+        " of running kernel: {}")},
+    {ProblemRules::RULE_MODULE_DISTUPGRADE, M_("{} does not belong to a distupgrade repository")},
```

Comment:
Just a question - what about information about repository?

Reply:
The changes do not prevent to add it in the future, so I though I would leave it for another PR to not make this one too big. But the disadvantage is that the strings for translations will change. I can take a look.